pwa/src/components/actionButton/ActionButton.tsx

```diff
@@ -62,8 +63,12 @@ export const ActionButton: React.FC<ActionButtonProps> = ({ actions, size = "md"
       <div className={clsx(styles.actionsContainer, isOpen && styles.isOpen)}>
         <ul>
           {actions.map((action, idx) => (
-            <li key={idx} onClick={(e) => handleActionClick(e, action)} className={action.disabled && styles.disabled}>
-              <FontAwesomeIcon icon={getIconFromBulkAction(action)} /> {_.upperFirst(action.type)}
+            <li
+              key={idx}
+              onClick={(e) => handleActionClick(e, action)}
+              className={clsx(action.disabled && styles.disabled, action.label && styles.buttonLabel)}
```

Comment:
I think it'd be better to add this `styles.buttonLabel` regardless of the label being passed. Because the styling applied here should also be applied to the standard label (based on `action.type`).

Reply:
If one of the options has an custom label and is longer than the standard ones the hover shadow will fail.

With `style.buttonLabel` applied only if label
![image](https://github.com/ConductionNL/gateway-ui/assets/43807324/f89acd90-db12-4f01-84a4-516a04e8cddb)

With `style.buttonLabel` applied to all
![image](https://github.com/ConductionNL/gateway-ui/assets/43807324/d14cd8aa-c090-4aaf-b572-08784e19af94)